### DISCUSSION TITLE
QEditorSlots Typescript Type Empty

### DISCUSSION THREAD

**Discussion by edwardmjackson** [quasarframework/quasar#16067] [General - Components / Directives / etc] [answered]
`quasar\dist\types\index.d.ts`  has an empty object defined for the slots on QEditor (see below). This causes a typescript error when trying to use a toolbar slot.

```
export interface QEditorSlots { }
```

Should it be something more like this: 

```
export interface QEditorSlots {
    /**
   * Slot used for expansion item's content
   */
    token: () => VNode[];
 }
 ```
 
to support the example in the [docs](https://quasar.dev/vue-components/editor#example--using-toolbar-slots) showing how to use a slot called token. Or am I missing something?

**Comment by yusufkandemir** [comment, marked as answer]
It shouldn't be exactly like that, it should be completely dynamic, token is just an example. It is resolved in #16069 and will be included in the next `quasar` release.
